paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/cryptography/DigesterConfiguration.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.server.configrue.cryptography;
+
+import org.apache.paimon.web.cyptography.DigesterService;
+import org.apache.paimon.web.cyptography.DigesterServiceFactory;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Objects;
+import java.util.Properties;
+import java.util.ServiceLoader;
+
+/** DigesterConfiguration. */
+@Getter
+@Setter
+@Slf4j
+@Configuration
+@ConfigurationProperties(prefix = "paimon.web.cryptography.digester")
+public class DigesterConfiguration {
+
+    private String algorithm;
+
+    private Properties properties;
+
+    @Bean
+    public DigesterService digesterService() {
```

Comment:
Why does this method define in `DigesterConfiguration`? Meanwhile, why does `NoneDigesterService` have no factory? Please correctly use factory mode.